index.html

```diff
@@ -40,31 +39,185 @@ <h1 class="heading_second">
+          <ul class="tech_list">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button class="btn1" type="submit">See Project</button>
+        </div>
+      </div>
+      <div class="work_card">
+        <div class="img_cardiv">
+          <img class="mobi" src="./images/thirdimage.png" alt="card" id="mobi-img3" />
+        </div>
+        <div class="side_info">
+          <h3 class="project_name">
+            Multi-Post<br>
+            Stories
+          </h3>
+          <ul class="first_list">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="tech_list">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button class="btn1" type="submit">See Project</button>
+        </div>
+      </div>
+      <div class="work_card">
+        <div class="img_cardiv">
+          <img class="mobi" src="./images/fourthimage.png" alt="card" id="mobi-img4" />
+        </div>
+        <div class="side_info">
+          <h3 class="project_name">
+            Multi-Post<br>
+            Stories
+          </h3>
+          <ul class="first_list">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="tech_list">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button class="btn1" type="button">See Project</button>
+        </div>
+      </div>
+    </div>
+    <section id="secondlast_section">
+      <div class="section-last-conatiner">
+        <div class="home_containe">
+          <div>
+            <h1 class="aboutdiv">About<br>Myself</h1>
+          </div>
+          <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my
+            work and experience! If you like what you see and have a project you need coded, don't hestiate to contact
+            me.
+          </p>
+          <span>LET'S CONNECT</span>
+          <ul class="social_media">
+            <li><a href="https://twitter.com/silviatofana"><img src="./images/twitter.svg" alt="twitter"></a></li>
+            <li><a href="https://www.linkedin.com/in/silviatofana"><img src="./images/Linkedin_icon.svg"
+                  alt="linkedin"></a>
+            </li>
+            <li><a href="#"><img src="./images/images4.svg" alt="ok"></a></li>
+            <li><a href="#"><img src="./images/images5.svg" alt="github"></a></li>
+            <li><a href="#"><img src="./images/images6.svg" alt="medium"></a></li>
+          </ul>
```

Comment:
- [ ] [Optional] I strongly suggest you use an icons library to make this kind of element easy to create, you can find social icons for example this [LinkedIn icon](https://fontawesome.com/search?p=1&q=linkedin&s=solid%2Cbrands)